packages/client/src/private-functions/cosmos/executeCosmosTransaction.ts

```diff
@@ -109,11 +110,15 @@ export const executeCosmosTransaction = async ({
   });
 
   const txBytes = TxRaw.encode(rawTx).finish();
+  const rawTxBase64 = Buffer.from(txBytes).toString("base64");
 
-  const txResponse = await stargateClient.broadcastTx(txBytes);
+  const txResponse = await submit({
+    chainId,
+    tx: rawTxBase64,
+  })
```

Comment:
i'm curious if we use submit endpoint how long does it take for the keplr tx window to popup compared to before where we pre-loaded the `stargateClient`